LSE-61.tex

```diff
@@ -3642,6 +3642,786 @@ \subsubsection{No Limit on Data Access Centers}
+} \\\hline
+\end{parameters}
+
+\subsubsection{Simultaneous users for high-volume queries}
+
+\label{DMS-REQ-0361}
+\textbf{ID:} DMS-REQ-0361 (Priority: 1b)
+
+\textbf{Specification:}
+The system shall support \textbf{hvQueryUsers }simultaneous high-volume queries running at any given time.
+
+\begin{parameters}
+Minimum number of simultaneous users performing high volume queries.
+&
+50
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+hvQueryUsers
+} \\\hline
+\end{parameters}
+
+\subsubsection{Real time query performance}
+
+\label{DMS-REQ-0356}
+\textbf{ID:} DMS-REQ-0356 (Priority: 1b)
+
+\textbf{Specification:}
+Low volume queries shall respond within \textbf{lvQueryTime} under a load of \textbf{lvQueryUsers} simultaneous queries.
+
+\textbf{Discussion:}
+A large fraction of ad-hoc user access will involve so called “low-volume” queries – queries that touch small area of sky, or request small number of objects.
+
+\begin{parameters}
+Maximum time allowed for retrieving results of a low-volume query.
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+lvQueryTime
+} \\\hline
+Minimum number of simultaneous users performing low volume queries.
+&
+100
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+lvQueryUsers
+} \\\hline
+\end{parameters}
+
+\subsubsection{Level 1 Database Query Performance}
+
+\label{DMS-REQ-0355}
+\textbf{ID:} DMS-REQ-0355 (Priority: 1b)
+
+\textbf{Specification:}
+The live Prompt Products Database shall support at least \textbf{l1QueryUsers} simultaneous queries, assuming each query lasts no more than \textbf{l1QueryTime}.
+
+\begin{parameters}
+Minimum number of simultaneous users querying the Prompt Products Database.
+&
+20
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+l1QueryUsers
+} \\\hline
+Maximum time allowed for retrieving results of a query of the Prompt Products Database.
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+l1QueryTime
+} \\\hline
+\end{parameters}
+
+\subsubsection{DM EFD Query Performance}
+
+\label{DMS-REQ-0358}
+\textbf{ID:} DMS-REQ-0358 (Priority: 1a)
+
+\textbf{Specification:}
+The DM copy of the EFD shall support at least \textbf{dmEfdQueryUsers} simultaneous queries, assuming each query lasts no more than \textbf{dmEfdQueryTime}.
+
+\begin{parameters}
+Maximum time allowed for retrieving results of a DM EFD query.
+&
+10
+&
+\unitname{%
+second
+}
+&
+\paramname{%
+dmEfdQueryTime
+} \\\hline
+Minimum number of simultaneous users querying the DM EFD.
+&
+5
+&
+\unitname{%
+integer
+}
+&
+\paramname{%
+dmEfdQueryUsers
+} \\\hline
+\end{parameters}
+
+\subsection{Image Access Performance Metrics}
+
+\subsubsection{Retrieval of postage stamp light curve images}
+
+\label{DMS-REQ-0375}
+\textbf{ID:} DMS-REQ-0375 (Priority: 2)
+
+\textbf{Specification:}
+Postage stamp cutouts, sized to match those in alert packets, of all observations of a single Object shall be retrievable within \textbf{postageStampRetrievalTime}, with \textbf{postageStampRetrievalUsers} simultaneous requests.
```

Comment:
Alert postage stamps were sized at 30x30.

Reply:
I picked the alert packet size since that would mean our postage stamps were consistent with our alerts. @gpdf please pick a number and I'll use it. I still don't know why we use an even number of pixels on a side though.